templates/prusa-connect-rtsp.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>prusa-connect-rtsp</Name>
+  <Repository>ghcr.io/beccazero/prusa-connect-rtsp:main</Repository>
+  <Registry/>
+  <Network>bridge</Network>
+  <MyIP/>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support/>
+  <Project>https://github.com/beccazero/prusa-connect-rtsp</Project>
+  <Overview>Capture images from a RTSP camera upload them to Prusa Connect.</Overview>
+  <Category>HomeAutomation:</Category>
+  <WebUI/>
+  <TemplateURL/>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/prusa-connect-rtsp.png</Icon>
+  <ExtraParams/>
+  <PostArgs/>
+  <CPUset/>
```

Comment:
There's a lot of unused variables in here that got auto-added by the plugin you likely used to convert an installed Docker container into an Unraid template. Please clean them up :)